en/flight_controller/crazyflie2.md

```diff
@@ -161,10 +163,24 @@ Then, you need to stick the battery on top of the SD card deck using a double si
 | IMU_DTERM_CUTOFF  | 70                          |
 | SYS_FMU_TASK      | Enabled                     |
 
-\* Since we haven't been able to fly it with optical flow, we don't use optical flow data yet.
+## Altitude Control 
 
+Crazyflie is able to fly in Altitude mode if you use a [Z-ranger deck](https://store.bitcraze.io/collections/decks/products/z-
+ranger-deck), connected to the board on I2C interface. The driver for this range finder is called "vl53lxx.cpp" which publishes 
```

Comment:
The driver is called vl53lxx - the source file is called vl53lxx.cpp. 
I am not sure that this level of detail is relevant - uORB is a developer topic. Unless they need to start this themselves,  I'd say all they need to now is the bit that starts "The maximum height ..."

Reply:
Both drivers are started automatically. 
The max height for the distance sensor is 2m (according to the datasheet), but we get 0.5m on a dark floor and 1.3m on a light floor. 
The max height for optical flow is the same as the distance sensor since we need the height in order to use flow.

Reply:
By the way are we planning to have a page also in the dev guide?

Reply:
@DanielePettenuzzo should I remove the driver info? 

> The max height for the distance sensor is 2m (according to the datasheet), but we get 0.5m on a dark floor and 1.3m on a light floor.
The max height for optical flow is the same as the distance sensor since we need the height in order to use flow.

I think it is important to make this distinction clear from the datasheet so I'll add this.

Reply:
> By the way are we planning to have a page also in the dev guide?

I don't think so, since we don't have it for other platforms. Should I add crazyflie's optical flow driver info [here](https://dev.px4.io/en/tutorials/optical_flow.html)?

Reply:
> By the way are we planning to have a page also in the dev guide?

Please don't. The reason you would put something in the dev guide is if this was a developer-only platform that required developer understanding to use. 

It is OK to have some developer information in the user guide - for example, if you have to build firmware to use the platform then you'd include the build command and link to the devguide for other information: https://docs.px4.io/en/flight_controller/mindpx.html#how-to-build

Generally speaking though, the developer information should be clearly identified as such. The user guide is primarily for platforms for which a somewhat tuned airframe exists. The dev guide is for creating new frame times and extending the stack itself. 

@barzanisar You should remove the driver info unless it is needed/useful for the audience. If it is useful then it would be better if the reasoning was explained and separated into a developer specific section. In either case I'd name the driver not the source file.

Reply:
@hamishwillee I think the driver names and topic it publishes to is just for our reference. We don't expect user to know about this, so I'll remove them.